estate/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate property"
+    name = fields.Char('Property Name', required=True)
```

Comment:
We don't often add useless empty line, but here it would help distinguish  
```suggestion
    _description = "estate property"

    name = fields.Char('Property Name', required=True)
```